BackEnd/Server/server.js

```diff
@@ -1,10 +1,50 @@
 const express = require("express");
+const ChatGpt = require("../../FrontEnd/src/Components/CardData/ChatGpt.json");
+const Midjourney = require("../../FrontEnd/src/Components/CardData/Midjourney.json");
+const DallE = require("../../FrontEnd/src/Components/CardData/DallE.json");
+const StableDiffusion = require("../../FrontEnd/src/Components/CardData/StableDiffusion.json");
+
+const ChatGptPath = "../../FrontEnd/src/Components/CardData/ChatGpt.json";
+const MidjourneyPath = "../../FrontEnd/src/Components/CardData/Midjourney.json";
+const DallEPath = "../../FrontEnd/src/Components/CardData/DallE.json";
+const StableDiffusionPath =
+  "../../FrontEnd/src/Components/CardData/StableDiffusion.json";
 
 const app = express();
 const port = 4002;
+app.use(express.json());
+
+app.get("/api/prompts/chatgpt", (req, res) => {
+  res.json(ChatGpt);
+});
+
+app.get("/api/prompts/midjourney", (req, res) => {
+  res.json(Midjourney);
+});
+
+app.get("/api/prompts/stablediffusion", (req, res) => {
+  res.json(StableDiffusion);
+});
+
+app.get("/api/prompts/dalle", (req, res) => {
+  res.json(DallE);
+});
+
+app.get("/api/prompt/:id", async (req, res) => {
+  const { id } = req.params;
+
+  try {
+    const prompt = await Prompt.findById(id);
```

Comment:
`Prompt` is not import/initialized in the file, isn't this throwing any error?

Reply:
Yes it does, I'm still working on it